### ISSUE TITLE
Jaws of the Lion Abilities > 4

### ISSUE THREAD

**Issue by orphu** [orichalque/yet-an-other-gloomhaven-companion#137]
The ability cards for Jaws of the Lion characters seem to be missing for any cards over level 4.

**Comment by orichalque** [comment]
Hi, we have no fix for this right now.
We used the cards from this repo: https://github.com/any2cards/jotl 
And their max level is 4 :/

**Comment by orphu** [comment]
I noticed gloom cards (https://github.com/ksokhan/gloom-cards) seems to have all of the JOTL cards, and also say they use any2cards as a source. But I can't seem to find them at any2cards!

**Comment by orphu** [comment]
I must have been blind. THey do have them. For example here's level 8 card for the Redguard.

**Comment by orichalque** [comment]
haha, happened to me too. They are sorted by name so we have to check the image to know the level :D

**Comment by orphu** [comment]
I'd be glad to help out here. Open for a PR?

**Comment by orichalque** [comment]
With pleasure :D

**Comment by orphu** [comment]
Quick question:

What does "points" entail in the card definitions? It appears to be a unique number for each card?

**Comment by orichalque** [comment]
I have honestly no idea, it was already in the any2cards database so we just kept it

**Comment by orphu** [comment]
Submitted #138
